src/libstore/http-binary-cache-store.cc

```diff
@@ -9,6 +9,15 @@ namespace nix {
 MakeError(UploadToHTTP, Error);
 
 
+StringSet HttpBinaryCacheStoreConfig::uriSchemes()
+{
+    static bool forceHttp = getEnv("_NIX_FORCE_HTTP") == "1";
```

Comment:
Is this new? Where is this environment variable used?

Reply:
It's not new, it was in the header before.

I think it is just used for tests

Reply:
Found in the checkout. Was just confused about the diff.